scripts/select_msoas.py

```diff
@@ -1,12 +1,37 @@
 # This script helps you define new study areas. You can write the output to a
-# new file, config/some_region.csv .
-
+# new file, config/some_region.csv.
 import csv
+import click
+
+@click.command()
+@click.option('--lda',
+             default='e.g Liverpool',
+             prompt = "Name of Local Authority District",
+             help="Enter the name of the Local Authority District of your preference, to get the MSOA codes for that area")
+
+def area(lda):
+    newarea= []
+    # You must run the SPC pipeline to download this file
+    click.echo("Querying MSOA codes for, %s!" % lda)
+    newFileNameLocation = "config/"+lda+".csv"
+    def new_dict(d, cols):
+        new_dict = {}
+        for col in cols:
+            new_dict[col] = d[col]
+        return new_dict
+
+    with open('data/raw_data/referencedata/lookUp.csv', 'r') as f:
+        df = csv.DictReader(f)
 
-print('"MSOA11CD"')
-# You must run the SPC pipeline to download this file
-with open('data/raw_data/referencedata/lookUp.csv') as f:
-    for row in csv.DictReader(f):
-        # Modify these to select MSOAs based on different geographies
-        if row['LAD20NM'] == 'Liverpool':
-            print('"{}"'.format(row['MSOA11CD']))
+        with open(newFileNameLocation, 'w') as csvfile:
+            fieldnames = ["MSOA11CD"]
+            writer = csv.DictWriter(csvfile, fieldnames=fieldnames, quoting=csv.QUOTE_NONNUMERIC)
```

Comment:
This is a very complicated way to print a CSV file with one column. Why not keep the old approach and just do `csvfile.write('"{}"'.format(row['MSOA11CD']))` or similar?

Reply:
Well this is the way I know, if you think is way complicated, then lets just make it as you've suggested. The important part for me is to write it in a CSV file directly.